internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -797,9 +797,18 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	var publishers template.PublishOpts
 	// convert the topics to template Topics
 	for _, topic := range topics {
+		var fifo template.Fifo
+		if !topic.Fifo.IsEmpty() {
+			if aws.BoolValue(topic.Fifo.Enable) {
+				fifo.Enable = aws.BoolValue(topic.Fifo.Enable)
+			}
```

Comment:
can be simplified to:
```suggestion
	fifo.Enable = aws.BoolValue(topic.Fifo.Enable)
```